.github/workflows/ci.yml

```diff
@@ -251,17 +256,36 @@ jobs:
           if [[ $GITHUB_REPOSITORY =~ /(silverstripe-spellcheck|recipe-authoring-tools)$ ]] || [[ "${{ matrix.phpunit_suite }}" == "recipe-authoring-tools" ]]; then
             sudo apt install -y hunspell libhunspell-dev hunspell-en-us
           fi
-          if [[ $GITHUB_REPOSITORY =~ /(silverstripe-dynamodb)$ ]]; then
+          if [[ $GITHUB_REPOSITORY =~ /(silverstripe-dynamodb)$ ]] && [[ $MATRIX_PHPUNIT == "true" ]]; then
             # install dynamodb
+            # https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html
             mkdir _dynamodb
             cd _dynamodb
-            sudo wget https://s3.ap-southeast-1.amazonaws.com/dynamodb-local-singapore/dynamodb_local_latest.tar.gz
-            sudo tar -xf dynamodb_local_latest.tar.gz
+            wget https://d1ni2b6xgvw0s0.cloudfront.net/v2.x/dynamodb_local_latest.tar.gz
+            wget https://d1ni2b6xgvw0s0.cloudfront.net/v2.x/dynamodb_local_latest.tar.gz.sha256
+            if [[ $(shasum -a 256 dynamodb_local_latest.tar.gz) != $(cat dynamodb_local_latest.tar.gz.sha256) ]]; then
+              echo "shasum for dynamodb_local_latest.tar.gz does not match"
+              exit 1
+            fi
+            tar -xvzf dynamodb_local_latest.tar.gz
+
             # run dynamo db as background process
             # java openjdk 11 is pre-installed
             $(java -Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar) &
-            for i in {1..10}; do
-              if ! [[ $(ps -aux | grep DynamoDBLocal.jar) =~ DynamoDBLocal_lib ]]; then
+            for i in {1..20}; do
```

Comment:
Why change this to 20?

Reply:
Just give it a little extra time in case the (shared) CPU is getting thrashed for whatever reason, no real downside.